### ISSUE TITLE
Implement additional Tracker get/set services for batch-processing and past-frame-reporting

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#539]
New services to be added: Note, properties must be supported by the low-level library. 
```C
/**
 * @brief Gets the current "enable-batch-process" settings for the named KTL 
 * or IOU Tracker object. 
 * @param[in] name unique name of the Tracker to query
 * @param[out] true if batch-processing is enabled, false otherwise
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_batch_processing_enabled_get(const wchar_t* name, 
    boolean* enabled);

/**
 * @brief Sets the "enable-batch-process" settings for the named KTL 
 * or IOU Tracker
 * Note: This call is only effective if the low-level library supports 
 * both batch and per-stream processing.
 * @param[in] name unique name of the Tracker to query
 * @param[out] true to enable batch-processing enabled, false otherwise
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_batch_processing_enabled_set(const wchar_t* name, 
    boolean enabled);

/**
 * @brief Gets the current "enable-past-frame" settings for the named KTL 
 * or IOU Tracker
 * @param[in] name unique name of the Tracker to query
 * @param[out] true if past-frame reporting is enabled, false otherwise.
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_past_frame_reporting_enabled_get(const wchar_t* name, 
    boolean* enabled);

/**
 * @brief Sets current "enable-past-frame" settings for the named KTL 
 * or IOU Tracker object
 * Note: This call is only effective if the low-level library supports 
 * past frame reporting.
 * @param[in] name unique name of the Tracker to query
 * @param[out] true if past frame reporting is enabled, false otherwise.
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_past_frame_reporting_enabled_set(const wchar_t* name, 
    boolean enabled);

```

**Comment by rjhowell44** [comment]
Merged into v0.16.alpha

**Comment by rjhowell44** [comment]
These services will be updated to DCF specific function..

**Comment by rjhowell44** [comment]
Merged into v0.16.alpha
